Products/ZenHub/zenhubworker.py

```diff
@@ -72,7 +72,7 @@ def __init__(self):
         loadPlugins(self.dmd)
         self.pid = os.getpid()
         self.services = {}
-        factory = ReconnectingPBClientFactory(pingPerspective=False)
+        factory = ReconnectingPBClientFactory(pingPerspective=self.options.ping_perspective)
         self.log.debug("Connecting to %s:%d",
```

Comment:
We don't want the ping perspective to ever be true for zenhub workers.  Why is this being made configurable?